stainless_extraction/src/expr/mod.rs

```diff
@@ -0,0 +1,358 @@
+use std::convert::TryFrom;
```

Comment:
It's a bit annoying, even though I `git mv` this file in the commit c35a44af19196e3ebfddbff40f701bc05b943cac it still shows up as deleted and added in the summary here...